deployments/ansible/molecule/custom_vars/windows-converge.yml

```
@@ -12,7 +12,6 @@
     splunk_otel_collector_config: '{{ansible_env.ProgramData}}\Splunk\OpenTelemetry Collector\custom_config.yml'
     splunk_otel_collector_config_source: ./custom_collector_config.yml
     splunk_memory_total_mib: 256
```

Test the new option on windows:
```suggestion
    splunk_memory_total_mib: 256
    go_mem_limit: 230
```

---

done